tinkerbell/stack/values.yaml

```diff
@@ -12,6 +12,20 @@ stack:
   lbClass: kube-vip.io/kube-vip-class
   # Once the Kubernetes Gateway API is more stable, we will use that for all services instead of nginx.
   image: nginx:1.25.1
+  persistence:
```

Comment:
if i understand correctly, persistence is only related to the HookOS download? if that is the case, would this block be more ideal under `hook:`?